ui/src/main/javascript/app/core/choosewonder/choosewonder.service.js

```diff
@@ -4,29 +4,29 @@ angular.
     module('sevenWonders.core.choosewonder').
     factory('ChooseWonder', ['$cookies', '$websocket', 'Game', 'Restangular', '$q', '$location',
         function ($cookies, $websocket, Game, Restangular, $q, $location) {
-                return {
-                    setWonderPlayer : function(player){
-                        return $q(function(resolve, reject){
+            return {
+                setWonderPlayer: function (player) {
+                    return $q(function (resolve, reject) {
                         Restangular.all('games/' + Game.getCurrentGame().id + '/player').customPUT(player)
-                                   .then(function(data) {
-                                         resolve(data);
-                                   }).catch(function(data){
-                                         reject(data);
-                                   });
-                       });
-                    },
+                            .then(function (data) {
+                                resolve(data);
+                            }).catch(function (data) {
+                                reject(data);
+                            });
+                    });
+                },
 
-                    getWonderPlayers : function () {
-                       return $q(function(resolve, reject){
+                getWonderPlayers: function () {
```

Comment:
I understand that this name is not general, we are here getting all the players of a specific game. This should have another name and not getWonderPlayers. I think it ...